packages/lodestar-beacon-state-transition/src/lightclient/naive/sync_committee.ts

```diff
@@ -73,7 +76,7 @@ export function processSyncCommittee(
  */
 export function getSyncCommitteeIndices(
   config: IBeaconConfig,
-  state: lightclient.BeaconState,
+  state: lightclient.BeaconState | phase0.BeaconState,
```

Comment:
Can you use `import {allForks} from "@chainsafe/lodestar-types";` `allForks.BeaconState` (and `allForks.BeaconBlock`) in all these places? Trying to use that everywhere for "multi-fork" types.

Feel free to remove `phase1.BeaconState` from `allForks.BeaconState`, for now, if it causes trouble.